recipes-multimedia/alsa/alsa-lib_%.bbappend

```diff
@@ -12,4 +12,6 @@ SRC_URI:append:imx-nxp-bsp = "${IMX_PATCH}"
 PACKAGE_ARCH:imx-nxp-bsp = "${MACHINE_SOCARCH}"
 
 GLIBC_64BIT_TIME_FLAGS = ""
+GLIBC_64BIT_TIME_FLAGS:arm:imx-nxp-bsp = " -D_TIME_BITS=64 -D_FILE_OFFSET_BITS=64"
```

Comment:
If you're not mistaken, there is a feature that is used to include or not include this support. If that's the case, then we should make this conditional to this distro feature and not architecture-specific.